tools/plugins/install_go_tools.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -x
+
+. /vagrant/conf.sh
+
+set -e
+
+GOPATH=/home/$VAGRANT_USER/go
+GOBIN=$GOPATH/bin
+
+export GOPATH
+echo "export GOPATH=$GOPATH:/project/sawtooth-core/sdk/go" >> /home/$VAGRANT_USER/.bashrc
+echo "export PATH=$PATH:$GOBIN" >> /home/$VAGRANT_USER/.bashrc
```

Comment:
This clobbers PATH since $PATH is expanded here.  Suggest adding GOPATH, PATH statements to local-env.sh instead.